A simple implementation of "Paste from clipboard" feature.

demield opened the pull request:
Ehm, hello. I've made a simple solution for "paste from clipboard" feature mentioned in #111. It's not really complete, need to add some proper messages for the user if something goes wrong (invalid mime type or puzzle string format). I would like to get some feedback on this.

demield committed:
A simple implementation of "Paste from clipboard" feature.

Now you can paste a puzzle string in sudoku puzzle editor.

ogarcia commented:
@demield Is a great idea. Simple and effective. If you add the inverted function to  "copy to clipboard" and the messages I'll merge it. Thanks!

demield committed:
Added simple implementation of "Copy to clipboard" feature, modified serialization, added some documentation.
Modified serialization methods to be able to convert CellCollection to plain text format (81 character string).
Wrote some documentation for serialization methods in CellCollection and Cell.

demield commented:
Hello, again. I've added "copy to clipboard" feature and modified serialization methods to be able to copy puzzle as plain text. If I understood correctly, currently serialization supports only version 3 format, so I mentioned it in a documentation.

demield committed:
Removed a typo, didn't know what to use as a ClipData label.
I think, it doesn't really matter.

ogarcia commented:
Merged! No more time for now, in next hours I'll publish a new release :wink:
